src/app/header/navuser.component.spec.ts

```diff
@@ -18,19 +19,19 @@ describe('NavUserComponent', () => {
       expect(el.querySelector('spinner')).toBeNull();
       expect(el.querySelector('.name').textContent).toEqual('TheAccountName');
       expect(navuser.userName).toEqual('TheAccountName');
-      expect(navuser.userImage).toEqual('http://cms.mock/v1/theimage');
+      expect(el.querySelector('image-loader > img').src).toEqual('http://cms.mock/v1/theimage');
```

Comment:
It's hard not to violate "separation of concerns" in these component tests.  But any test is better than nothing!